packages/react/package.json

```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "@gadgetinc/api-client-core": "^0.15.11",
     "react-fast-compare": "^3.2.2",
-    "react-hook-form": "^7.46.1",
+    "react-hook-form": "~7.46.1",
```

Comment:
Is it worth pinning this to `~7.48.2` to get latest before the node `v18` change?

Reply:
:+1: